http4s-blaze-client/src/main/scala/com/avast/server/toolkit/http4s/Http4sBlazeClientConfig.scala

```diff
@@ -0,0 +1,30 @@
+package com.avast.server.toolkit.http4s
+
+import java.util.concurrent.TimeUnit
+
+import com.avast.server.toolkit.ssl.SslContextConfig
+import org.http4s.BuildInfo
+import org.http4s.client.blaze.ParserMode
+import org.http4s.client.defaults
+import org.http4s.headers.{`User-Agent`, AgentComment, AgentProduct}
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+final case class Http4sBlazeClientConfig(
```

Comment:
I know I've said it before, but I just want to say again how happy having case classes for configuration makes me! :smile_cat:

Reply:
💪 👍